atomic_reactor/schemas/user_params.json

```diff
@@ -31,6 +31,7 @@
     "koji_task_id": {"type": "integer"},
     "koji_upload_dir": {"type": "string"},
     "name": {"type": "string"},
+    "parent_images_digests": {"type": "object"},
```

Comment:
type: object covers a lot of structures.  Can you provide more detail on the internals of this structure here?

Reply:
I can update schema, but It will be too generic anyway.

Example of data
```
{
  'registry/fedora:latest': {
    'x86_64': 'fedora@sha256:14654aaf....',
    'ppc64le': 'fedora@sha256:445431dea...'
  }
}
```

Race condition is eliminated because pull_base_image will use the same digests as orchestrator, workers are unaffected by image tag change during build.

Reply:
I feel like we only need a single manifest digest (the one for the manifest list) for each image name. All the workers can use the same manifest list and find their own arch-native image within it.

Reply:
I've updated schema, please check if it's ok or if it should be more specific